Traceback (most recent call last):
  File "snippet.py", line 11, in <module>
    import credential_utils
ImportError: No module named credential_utils
